packages/tag/Tag.scss

```diff
@@ -1,3 +1,4 @@
 @import "../generics.shared";
 @import '~blaze/scss/components.buttons';
 @import '~blaze/scss/components.tags';
+@import '~blaze/scss/components.inputs';
```

Comment:
we should use existing compoments `bl-input`,`bl-button` so we dont have duplicate styles in various components